pkg/pillar/types/domainmgrtypes.go

```diff
@@ -39,6 +39,33 @@ type DomainConfig struct {
 
 	// CipherBlockStatus, for encrypted cloud-init data
 	CipherBlockStatus
+
+	// MetaDataType for select type of metadata service for app
+	MetaDataType MetaDataType
+}
+
+// MetaDataType of metadata service for app
+type MetaDataType uint8
+
+// types of metadata service for app
```

Comment:
Please add comment that this must match the values in the proto definition.

Reply:
Done